packages/webdriverio/src/constants.js

```diff
@@ -327,23 +328,41 @@ export const UNICODE_CHARACTERS = {
     'ArrowDown': '\uE015',
     'Insert': '\uE016',
     'Delete': '\uE017',
+    ';': '\uE018',
```

Comment:
Why do we need to define these twice?

Reply:
[there](https://webdriver.io/docs/api/browser/keys.html) is doc says:

> You’ll find all supported characters [here](https://w3c.github.io/webdriver/webdriver-spec.html#keyboard-actions). 

And there are these codes: `";", "=", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "*", "+", ",", "-", ".", "/",`.

I'm looking into `grapheme-splitter` and it it just split all and send such symbols as is.

Ok maybe, I should delete such symbols from PR.
I'll do, if you say.

Reply:
yes but these are all defined twice .. if you want to use these special characters you e.g. use `Semicolon` or just send `;` as a string.

Reply:
Maybe doc should be rewritten to other key list, instead of w3c one?

Reply:
Yes, we should link to the constant table. I honestly don't know what the difference between sending `Numpad 0` and `0`

Reply:
`Numpad<N>` should be a keycode for separate Numpad part of keyboard

W3C spec includes it too
https://w3c.github.io/webdriver/#keyboard-actions

It should return the same code as `<N>` key, but when shifted, the same as `Insert`